litgpt/deploy/serve.py

```diff
@@ -23,6 +31,12 @@ def __init__(self,
                  top_k: int = 50,
                  max_new_tokens: int = 50) -> None:
 
+        if not _LITSERVE_AVAILABLE:
+            raise ImportError(
+                "Serving LitGPT requires the `litserve` library,"
+                " which can be installed via `pip install litserve`."
+            )
```

Comment:
This will show a similar error and include any information for why it's not available

```suggestion
            raise ImportError(str(_LITSERVE_AVAILABLE))
```